website/webpack.config.js

```diff
@@ -46,7 +46,8 @@ const COMMON_CONFIG = {
 
   resolve: {
     alias: {
-      'luma.gl': libSources
+      'luma.gl': libSources,
+      'math.gl': join(__dirname, 'node_modules/math.gl')
```

Comment:
Not sure if this is the right way to expose math.gl to our examples, adding dependency in website/package.json only doesn't work, not super clear about the reason why it doesn't work.

Reply:
Hard to say what the problem is without testing. Maybe you could just add a TODO comment here about this, we can come back to it later.